frontend/packages/portal-frontend/src/compound/CompoundDoseViability.scss

```diff
@@ -0,0 +1,214 @@
+// Dose Curves
+
+.mainContentContainer {
+  margin-left: 10px;
+  margin-right: 60px;
+  padding-bottom: 60px;
+
+  .mainContentHeader {
+    margin-top: 20px;
+    margin-bottom: 20px;
+
+    p {
+      max-width: 780px;
+    }
+  }
+
+  .mainContentGrid {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    grid-template-areas: "plot plot selections";
+    gap: 1rem;
+
+    @media (max-width: 950px) {
+      grid-template-areas:
+        "selections"
+        "plot";
+      grid-template-columns: 1fr;
+      grid-template-rows: auto auto;
+    }
+  }
+
+  .mainContentHr {
+    border-top: 1px solid #b8b8b8;
+    margin-top: 30px;
+  }
+
+  .mainContentCellLines {
+    margin-top: 20px;
+    margin-bottom: 20px;
+
+    p {
+      max-width: 780px;
+    }
+  }
+}
+
+.PlotSection {
```

Comment:
It looks like this is very similar to the existing `CompoundDoseCurves.scss` file that's used by the existing dose curves tab. Ideally it would be nice to avoid the duplication though. Are you thinking that the older `CompoundDoseCurves.scss` file would be removed in the near future if the older dose curves tab is removed?

If not, do you think it would be possible to remove some of the duplication by moving this file somewhere that it can be used by both components?